src/App.js

```diff
@@ -35,13 +35,12 @@ export class App extends Component {
           <div className="o-container">
             <div className="c-site-content">
               <Switch>
-              <Route path="/add-form" component={AddForm} />
+                <Route path="/AddForm" component={AddForm} />
```

Comment:
The value of `path` prop does not have to match the name of a component. In this case you can use `/add-form` if you want :) It's not a problem but it looks like you assumed that those names have to match.